sdks/go/test/integration/integration.go

```diff
@@ -91,11 +91,7 @@ var portableFilters = []string{
 var flinkFilters = []string{
 	// TODO(BEAM-11500): Flink tests timing out on reads.
 	"TestXLang_Combine.*",
-	// TODO(BEAM-12815): Test fails: "Insufficient number of network buffers".
-	"TestXLang_Multi",
 	"TestDebeziumIO_BasicRead",
-	// TODO(BEAM-12753): Flink test stream fails for non-string/byte slice inputs
-	"TestTestStream.*Sequence.*",
 	// Triggers are not yet supported
 	"TestTrigger.*",
```

Comment:
We should try (in separate PRs) removing the other test filters as well, as I can see all of these being tightly related (timeouts and memory for debezium, and Triggers leaning on TestStream).